docs/raspberrypi.md

```diff
@@ -0,0 +1,14 @@
+# Notes on using Yoe on the Raspberry PI
+
+## Enable serial console
```

Comment:
actually we can say

add ENABLE_UART = "1" to local.conf

and that will do the needed configuration changes in image.